src/probe/probe.c

```diff
@@ -1010,6 +1010,8 @@ int probe_point_add(uint32_t count, struct probe_point *probe)
 				tracev_probe("probe_point_add(): start probe task");
 				schedule_task(&_probe->dmap_work, 1000, 1000);
 			}
+			/* ignore probe stream tag for extraction probes */
+			probe[i].stream_tag = _probe->ext_dma.stream_tag;
```

Comment:
Should we update the ABI to show it's unused ?

Reply:
No, it was ignored in-design for extraction, only valid for injection
To add probe point user have to enter 3x u32 and depending on purpose (extraction/injection): FW will use 2 or 3 of them
No changes for usage flow for user, kernel and FW, we will just not save this ignored field.

Reply:
@abonislawski the commit title and comment are misleading. It should say "Overriding the user-provided stream tag for extraction probe points"